libs-scala/gatling-utils/src/test/resources/simulation-log/minimal.txt

```diff
@@ -0,0 +1,4 @@
+ASSERTION	AAIBAAIFAAAAAAAAAAAA
+RUN	simulation.StandardFixTradeSimulation	standardfixtradesimulation	1587679986754	, args: --ledger vdaml-stub:6865 --ia-ttl 120sec --spec --simulation fix-trade.standard --trade-file /home/dlt/app/abcdef-load-tests/data/cde7/20191205_Trades.txt --trade-timeout 30s --concurrency 20 --bmw-timeout 20sec --requests 10000 --loop-file	3.1.1
+USER	fix-trade.standard	1	START	1587679986860	1587679986860
+REQUEST	1		fix-trade-sync	1587679987299	1587680007510	OK	ACK
```

Comment:
Sorry, still all these `fix-trade` things... 
How about a search-replace by a similar length? `s/FixTrade/TwoSteps/g` and `s/fix-trade/two-steps/g` for example.
And please delete all options that say  `--trade-blah`
That would give the following:
```suggestion
RUN	simulation.StandardTwoStepsSimulation	standardftwostepssimulation	1587679986754	, args: --ledger vdaml-stub:6865 --ia-ttl 120sec --spec --simulation two-steps.standard --concurrency 20 --bmw-timeout 20sec --requests 10000 --loop-file	3.1.1
USER	two-steps.standard	1	START	1587679986860	1587679986860
REQUEST	1		two-steps-sync	1587679987299	1587680007510	OK	ACK
```
The string replace should be very easy, so mainly just removal of the trade-related run arguments.